tmt/utils.py

```diff
@@ -309,8 +309,31 @@ class StructuredFieldError(GeneralError):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 def quote(string):
-    """ Surround a string with double quotes """
-    return f'"{string}"'
+    """ Quote given string in a shell-compatible way """
+    return shlex.quote(string)
+
+
+def quoted_env(env_vars):
+    """
+    Returns list of quoted of enviornment variables.
```

Comment:
```suggestion
    Returns list of quoted of environment variables.
```

Reply:
Removed, I do not need this function, it was a blind road